packages/client/src/ui/NetworkSelect.tsx

```diff
@@ -24,11 +24,21 @@ class NetworkSelectComponent extends React.Component<IProps, {}> {
   render(): JSX.Element {
     const enableThemedStyles = !this.onLandingPage();
     const color = enableThemedStyles ? undefined : COLORS.DARK_TITLE;
+
+    const filteredAvailableNetworks = () => {
+      return Object.values(AVAILABLE_NETWORKS).filter(
```

Comment:
Can we not just change AVAILABLE_NETWORKS here?

Reply:
Yes we can, I did not disturb it to prevent breaking unintended stuffs accidentally.